server/data/curiousClient.ts

```diff
@@ -7,15 +7,23 @@ export default class CuriousClient {
     return new RestClient('Curious API Client', config.apis.curious, token)
   }
 
-  async getPrisonerProfile(prisonNumber: string, token: string): Promise<Array<LearnerProfile>> {
-    return (await CuriousClient.restClient(token).get({
+  async getLearnerProfile(prisonNumber: string, establishmentId: string, token: string): Promise<LearnerProfile> {
+    const learnerProfiles = (await CuriousClient.restClient(token).get({
       path: `/learnerProfile/${prisonNumber}`,
-    })) as Promise<Array<LearnerProfile>>
+      query: `establishmentId=${establishmentId}`,
+    })) as Array<LearnerProfile>
+    return learnerProfiles.pop()
```

Comment:
The array will only contain one element, since we are passing in establishmentId. Therefore I think it makes no sense to return an array from the client.

Reply:
Changed back to an array of one or more elements.